### DISCUSSION TITLE
How to do Weighted Quantile Loss (WQL) on Chronos model using my own dataset

### DISCUSSION THREAD

**Discussion by Manukrish2504** [amazon-science/chronos-forecasting#201] [Q&A]

**Comment by lostella** [comment]
@Manukrish2504 one way is to adapt this evaluation script https://github.com/amazon-science/chronos-forecasting/blob/main/scripts/evaluation/evaluate.py: you can probably modify the `main` function there to load your dataset instead of instantiating it from a config file as it does.

**Comment by Manukrish2504** [comment]
So should I copy and add the main function into my code?  What about the config path and metric path...and where to give my dataset?
